Makefile.in

```diff
@@ -45,7 +45,7 @@ install:
 	install $(GOBIN)/api $(sbindir)/masapi
 	install -m 644 $(srcdir)/zoom.png $(datarootdir)
 	install -m 644 $(srcdir)/data_unavailable.png $(datarootdir)
-	for f in $(srcdir)/templates/* ; do install -m 644 $$f $(datarootdir)/gsky/templates ; done
+	cp -rp $(srcdir)/templates/* $(datarootdir)/gsky/templates
```

Comment:
Files should be installed with `install` where possible. What problem is this trying to solve?

Reply:
I tried `install` but it doesn't copy subdirectories. Originally WPS output template is hard coded to be a single template file. But after generalization, we will need different output templates for different clients. So I have subdirectory under `templates/` that stores different output templates.

Reply:
OK (you probably noted my comment to this effect in `Makefile.in`).